peripherals/spi/spi_v3.yaml

```diff
@@ -0,0 +1,281 @@
+    RCRCI:
+      AllZeros: [0, "All zeros RX CRC initialization pattern"]
+      AllOnes: [1, "All ones RX CRC initialization pattern"]
+    CRC33_17:
+      Disabled: [0, "Full size (33/17 bit) CRC polynomial is not used"]
+      Enabled: [1, "Full size (33/17 bit) CRC polynomial is used"]
+    SSI:
+      SlaveSelected: [0, "0 is forced onto the SS signal and the I/O value of the SS pin is ignored"]
+      SlaveNotSelected: [1, "1 is forced onto the SS signal and the I/O value of the SS pin is ignored"]
+    HDDIR:
+      Receiver: [0, "Receiver in half duplex mode"]
+      Transmitter: [1, "Transmitter in half duplex mode"]
+    CSUSP:
+      _write:
+        NotRequested: [0, "Do not request master suspend"]
+        Requested: [1, "Request master suspend"]
+    CSTART:
+      NotStarted: [0, "Do not start master transfer"]
+      Started: [1, "Start master transfer"]
+    MASRX:
+      Disabled: [0, "Automatic suspend in master receive-only mode disabled"]
+      Enabled: [1, "Automatic suspend in master receive-only mode enabled"]
+    SPE:
+      Disabled: [0, "Peripheral disabled"]
+      Enabled: [1, "Peripheral enabled"]
+
+  CR2:
+    TSER: [0, 65535]
+    TSIZE: [0, 65535]
+
+  CFG1:
+    MBR:
+      Div2: [0, "f_spi_ker_ck / 2"]
+      Div4: [1, "f_spi_ker_ck / 4"]
+      Div8: [2, "f_spi_ker_ck / 8"]
+      Div16: [3, "f_spi_ker_ck / 16"]
+      Div32: [4, "f_spi_ker_ck / 32"]
+      Div64: [5, "f_spi_ker_ck / 64"]
+      Div128: [6, "f_spi_ker_ck / 128"]
+      Div256: [7, "f_spi_ker_ck / 256"]
+    CRCEN:
+      Disabled: [0, "CRC calculation disabled"]
+      Enabled: [1, "CRC calculation enabled"]
+    CRCSIZE: [3, 31]
+    TXDMAEN:
+      Disabled: [0, "Tx buffer DMA disabled"]
+      Enabled: [1, "Tx buffer DMA enabled"]
+    RXDMAEN:
+      Disabled: [0, "Rx buffer DMA disabled"]
+      Enabled: [1, "Rx buffer DMA enabled"]
+    UDRDET:
+      StartOfFrame: [0, "Underrun is detected at begin of data frame"]
+      EndOfFrame: [1, "Underrun is detected at end of last data frame"]
+      StartOfSlaveSelect: [2, "Underrun is detected at begin of active SS signal"]
+    UDRCFG:
+      Constant: [0, "Slave sends a constant underrun pattern"]
+      RepeatReceived: [1, "Slave repeats last received data frame from master"]
+      RepeatTransmitted: [2, "Slave repeats last transmitted data frame"]
+    FTHVL:
+      OneFrame: [0, "1 frame"]
+      TwoFrames: [1, "3 frames"]
+      ThreeFrames: [2, "3 frames"]
+      FourFrames: [3, "4 frames"]
+      FiveFrames: [4, "5 frames"]
+      SixFrames: [5, "6 frames"]
+      SevenFrames: [6, "7 frames"]
+      EightFrames: [7, "8 frames"]
+      NineFrames: [8, "9 frames"]
+      TenFrames: [9, "10 frames"]
+      ElevenFrames: [10, "11 frames"]
+      TwelveFrames: [11, "12 frames"]
+      ThirteenFrames: [12, "13 frames"]
+      FourteenFrames: [13, "14 frames"]
+      FifteenFrames: [14, "15 frames"]
+      SixteenFrames: [15, "16 frames"]
+    DSIZE: [3, 31]
```

Comment:
Same comment as `CRCSIZE`: set to `[0, 31]`?

Reply:
ACK